extensions/kubernetes-config/runtime/src/main/java/io/quarkus/kubernetes/config/runtime/KubernetesConfigSourceFactory.java

```diff
@@ -84,7 +84,7 @@ Iterable<ConfigSource> getConfigSources(final KubernetesConfigSourceConfig confi
 
     private boolean isExplicitlyDisabled(ConfigSourceContext context) {
         ConfigValue configValue = context.getValue("quarkus.kubernetes-config.enabled");
-        if (AbstractRawDefaultConfigSource.NAME.equals(configValue.getConfigSourceName())) {
+        if ("DefaultValuesConfigSource".equals(configValue.getConfigSourceName())) {
```

Comment:
Should it be a constant? IS the other constant still in use?

Reply:
My understanding is that this will go away when the `TODO` is implemented

Reply:
Yes, I need to improve this a bit. This was a migration as is, so I didn't want to introduce other changes as well. As for the constant, the source is provided by SR Config, so I prefer to add the constant there.